content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -545,6 +545,65 @@ it.skip('returns "fizz" when number is multiple of 3', () => {
 })
 ```
 
+### Test Isolation
+
+<Alert type="success">
+
+<Icon name="check-circle" color="green"></Icon> **Best Practice:** Clean up
+state **before** tests run.
+
+</Alert>
+
+Test isolation is the practice of resetting application state _before_ each
+test.
+
+Cleaning up state ensures that the operation of one test does not affect another
+test later on. The goal for each test should be to reliably pass whether run in
+isolation or consecutively with other tests. Having tests that depend on the
+state of an earlier test can potentially cause nondeterministic test failures.
+
+Cypress supports two modes of test isolation, `lax` and `strict`.
+
+#### Lax Mode
+
+When in `lax` mode, Cypress handles resetting the state for:
+
+- [aliases](/api/commands/as)
+- [cookies](/api/commands/clearcookies)
+- [clock](/api/commands/clock)
+- [intercepts](/api/commands/intercepts)
+- [localStorage](/api/commands/clearlocalstorage)
+- [routes](/api/commands/route)
+- [sessions](/api/commands/session)
+- [spies](/api/commands/spy)
+- [stubs](/api/commands/stub)
+- [viewport](/api/commands/viewport)
+
+#### Strict Mode
+
+<Alert type="warning">
+
+<strong class="alert-header"><Icon name="exclamation-triangle"></Icon>
+Experimental</strong>
+
+Strict mode is currently experimental and can be enabled by setting
```

Comment:
Not sure if this a good suggestion since it seems grammatically out of place or if it keeps the strict reference consistent through the markdown? 🤷🏻 

```suggestion
`strict` mode is currently experimental and can be enabled by setting
```